VenueRendering/VenueRenderer.cs

```diff
@@ -90,7 +95,7 @@ public EmbedBuilder RenderEmbed(Venue venue, string bannerUrl = null, VenueRende
         if (resolution != null && resolution.IsAt(DateTimeOffset.Now))
             stringBuilder.AppendLine(":green_circle: **Open right now**");
         else if (resolution != null)
-            stringBuilder.Append(":black_circle: **Open ").Append(resolution.Start.DateTime.ToNow()).AppendLine("**");
+            stringBuilder.Append(":black_circle: **Open ").Append(resolution.Start.UtcDateTime.ToNow()).AppendLine("**");
```

Comment:
This is a fix to venue rendering showing the venue's opening in relative time incorrectly. Moment.net apparently takes the DateTime object it's given and compares that to the datetime as it is now but always in the host machines timezone, without any consideration to the timezone on the given DateTime.